packages/account/src/Components/poo-verified/verified.jsx

```diff
@@ -0,0 +1,13 @@
+import { Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import React from 'react';
+import IconMessageContent from 'Components/icon-message-content';
+
+export const POOVerified = () => {
```

Comment:
Hello @ali-hosseini-fs,

Because I export it in index.js as a default (index.js is in the same folder as that, so that folder is treated as a module and I don't have to add the extension)